newIDE/electron-app/app/Utils/LocalNetworkIpFinder.js

```diff
@@ -10,7 +10,10 @@ const getLocalNetworkIps = () => {
     .map(({ address }) => address);
 };
 
-/** @returns {?string} */
+/**
+ * Returns null if not connected to internet with an IPv4 connection.
+ * @returns {?string}
+ */
 const findLocalIp = () => {
```

Comment:
should we change this function to return 127.0.0.1 if ipAddresses === null ?

Reply:
I'm not sure if returning a local (in the scope of the machine) IP address is really expected from this function

Reply:
Then the name of the function is really confusing 😅 
Either way, lgtm 👍